tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/BinaryProjectImportWizardPage.java

```diff
@@ -78,7 +80,18 @@ protected void createBinaryLocationField( Composite parent )
         label.setLayoutData( new GridData( GridData.HORIZONTAL_ALIGN_BEGINNING ) );
 
         binariesLocation = SWTUtil.createSingleText( parent, 1 );
-
+      
```

Comment:
You will need to import our formatter preferences that are available in the liferay-ide repository under resources folder.  Also we use spaces instead of tabs so you can make that setting change in Eclipse as well.